keanu-python/tests/test_stats.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pytest
+
+from keanu.vertex import Uniform
+from keanu.algorithm import sample
+from keanu import Model, stats
+
+
+def test_can_get_autocorrelation():
+    x = np.arange(1, 9)
+    x_list = [np.array(a, float) for a in x]
+    autocorr = stats.autocorrelation(x_list)
+    expected = [1., 0.625, 0.27380952, -0.0297619, -0.26190476, -0.39880952, -0.41666667, -0.29166667]
+    np.testing.assert_almost_equal(autocorr, expected)
+
+
+def test_can_get_autocorrelation_for_samples():
+    with Model() as m:
+        m.gaussian = Uniform(0, 1000)
+    net = m.to_bayes_net()
+    samples = sample(net=net, sample_from=list(net.get_latent_vertices()), algo="metropolis", draws=1000)
+    valid_key = list(samples.keys())[0]
+    autocorr = stats.autocorrelation(samples.get(valid_key))
```

Comment:
can you assert something about `autocorr`?

Reply:
I could do a similar assertion that I do in the Java side. I assert that for a randomly generated sequence, the autocorrelation is close to 0. I am wondering it's worth repeating that test here though?